sugoi-api-ldap-store-provider/src/main/java/fr/insee/sugoi/store/ldap/LdapReaderStore.java

```diff
@@ -182,7 +174,7 @@ public SearchResultEntry getEntryByDn(String dn) {
   @Override
   public Group getGroup(String appName, String groupName) {
     SearchResultEntry entry = getGroupResultEntry(appName, groupName);
-    return (entry != null) ? GroupLdapMapper.mapFromSearchEntry(entry) : null;
+    return (entry != null) ? (new GroupLdapMapper()).mapFromSearchEntry(entry) : null;
```

Comment:
is it necessary to create this object every time ?

Reply:
I chose to make mapFromSearchEntry not static because in the generic method searchOnLdap with a static method I was forced to do complicated casts. But I wasn't very sure of that choice because I don't really know how expensive it is to instantiate this object every time

Reply:
ok, can we create this GroupLdapMapper only one time ? Maybe as a static attribute in the store ?